moped-etl/arcgis/components_to_agol.py

```diff
@@ -249,22 +254,19 @@ def main(args):
             features = handle_status_updates(features_of_type)
 
             logger.info("Deleting all existing features...")
-            delete_all_features(feature_type)
+            if not args.test:
+                delete_all_features(feature_type)
 
             logger.info(
                 f"Uploading {len(features)} features in chunks of {UPLOAD_CHUNK_SIZE}..."
             )
             for feature_chunk in chunks(features, UPLOAD_CHUNK_SIZE):
                 logger.info("Uploading chunk....")
-                add_features(feature_type, feature_chunk)
+                if not args.test:
+                    add_features(feature_type, feature_chunk)
     else:
-        # Get unique project IDs that need to have features deleted & replaced
-        project_ids = []
-
-        for component in data:
-            project_ids.append(component["project_id"])
-
-        project_ids_for_feature_delete = list(set(project_ids))
+        # Get project IDs have been updated (including soft-deleted projects) for deletes
```

Comment:
missing a word between IDs and have, "that" or "which" maybe?

Reply:
ooo thanks - i'll fix! 🙏